news/4461.bugfix

```diff
@@ -0,0 +1,2 @@
+This fix clarifies the error given to suggest to the user
+what might be wrong.
```

Comment:
This still isn't wright, sorry - you need to think about how it would look in the larger ```NEWS``` file. There's no context here explaining which error message has been changed, or 

What I was trying to suggest was that your original text:

> This fixes an issue where when someone who tries to use git with pip but pip can't because git is not in the path
environment variable. This explicitly tells them how they can fix the issue.

needed to be changed because the revised error no longer explicitly told the user how to fix the issue, but rather suggested to the user how to check what the problem might be. So the text I suggested was intended as a possible replacement for the last sentence only.